tests/pkg/assertions/assertions.go

```diff
@@ -519,6 +516,25 @@ func AssertMultipleEntitiesFromORDService(t *testing.T, respBody string, entitie
 	}
 }
 
+func AssertBundleCorrelationIds(t *testing.T, respBody string, entitiesMap map[string][]string, expectedNumber int) {
+	numberOfEntities := len(gjson.Get(respBody, "value").Array())
+	require.Equal(t, expectedNumber, numberOfEntities)
+
+	for i := 0; i < numberOfEntities; i++ {
+		entityTitle := gjson.Get(respBody, fmt.Sprintf("value.%d.title", i)).String()
+		require.NotEmpty(t, entityTitle)
+
+		correlationIds := gjson.Get(respBody, fmt.Sprintf("value.%d.correlationIds", i)).Array()
+		bundleCorrelationIdsAsString := make([]string, 0)
+		for _, id := range correlationIds {
+			idAsString := gjson.Get(id.String(), "value").String()
+			bundleCorrelationIdsAsString = append(bundleCorrelationIdsAsString, idAsString)
+		}
+		expectedCorrelationIds := entitiesMap[entityTitle]
+		assert.True(t, unorderedEqualStringArrays(expectedCorrelationIds, bundleCorrelationIdsAsString))
```

Comment:
here you can use the built-in `assert.ElementsMatch(t, expectedCorrelationIds, bundleCorrelationIdsAsString)`

you can double-check its documentation but from what I saw, the order of the given slices is ignored when comparing them